Topics/statistics-fundamentals.md

```diff
@@ -0,0 +1,167 @@
+# Statistics Fundamentals
+
+# What you will learn 
+
+In this guide, you'll learn what is the fundamental statistical knowledge you need to have in order
+to build insightful Data Science projects and build machine learning systems upon them.
+
+You'll also learn that not so much is needed to start to apply yourself to real-world data problems!
+
+## Prerequisites
+The previous Paradiso guides.
```

Comment:
Possibly add the "math fundamentals" guide to this? Or maybe mention that both should be taken care of "at the same time"?

Reply:
I would second that

Reply:
Back-linking paradiso or any other terms to the README page will only help and clarify to the readers, it won't take away anything that already is present in the guide.

Reply:
agree. done